.github/workflows/preview-build.yml

```diff
@@ -176,13 +177,14 @@ jobs:
         if: always() && steps.deployment.outputs.result
         env:
           PR_NUMBER: ${{ github.event.pull_request.number }}
+          FIRST_PAGE_PATH: ${{ steps.docs-build.outputs.first-page-path }}
```

Comment:
`first-page-path` => `landing-page-path` ? Maybe in the future we make this configurable in docset.yml or we have another mechanism for selecting the best page to advertise. 

Should we declare this in the action.yml for docs-builder too?

Reply:
Sorry, forgot to put this PR in draft. Nevertheless, thank you for the hint!

https://github.com/elastic/docs-builder/pull/642/commits/90820f133219bbde4d4458066ea97381e3f5829e